tests/py/requirements.txt

```diff
@@ -12,3 +12,6 @@ transformers==4.40.2
 # TODO @lanlao-nvidia Renable when modelopt can be install properly to run the tests
 # "nvidia-modelopt[all]">=0.16.1,<0.17.0
 --extra-index-url https://pypi.nvidia.com
+torchvision>=0.20.0.dev,<0.21.0
+--extra-index-url https://download.pytorch.org/whl/nightly/cu124
```

Comment:
I think this `extra-index-url` is redundant. We already have `INDEX_URL` defined in `install-torch-tensorrt.sh` which has templated channel and cu_version.

Reply:
yes, It is redundant.
in the install-torch-tensorrt.sh it is templated with channel and cu_version.
I still want to keep this --extra-index-url here just in case that developer run:
python -m pip install --pre -r tests/py/requirements.txt in their local environment to test